common/src/main/java/com/microsoft/identity/common/internal/cache/AccountCredentialCache.java

```diff
@@ -46,12 +45,15 @@
 
 import static com.microsoft.identity.common.internal.cache.CacheKeyValueDelegate.CACHE_VALUE_SEPARATOR;
 
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public class AccountCredentialCache implements IAccountCredentialCache {
 
     private static final String TAG = AccountCredentialCache.class.getSimpleName();
 
-    // The names of the SharedPreferences file on disk.
-    private static final String ACCOUNT_CREDENTIAL_SHARED_PREFERENCES =
+    /**
+     * The name of the SharedPreferences file on disk.
+     */
+    public static final String DEFAULT_ACCOUNT_CREDENTIAL_SHARED_PREFERENCES =
```

Comment:
One question, are we making all the constants in classes `public`?

Reply:
Not by design - in this case, I've made the default location `public` so that another class can specify the location instead of this class dictating it

Reply:
Part of [this other PR](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1225), which will be updated once all this merges.